scenarios/uturn/scenario.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pathlib import Path
+
+from smarts.sstudio import gen_scenario
+from smarts.sstudio import types as t
+
+
+traffic = t.Traffic(
+    flows=[
+        t.Flow(
+            route=t.Route(begin=("edge-west-EW", 0, 0), end=("edge-west-EW", 0, "max")),
+            rate=1,
+            actors={t.TrafficActor(name="car"): 1.0},
+        )
+    ]
+)
+
+social_agent_missions = {
+    "all": (
+        [
+            t.SocialAgentActor(
+                name="open-agent", agent_locator="open_agent:open_agent-v0"
+            ),
+        ],
+        [
+            t.Mission(
+                t.Route(begin=("edge-west-WE", 0, 50), end=("edge-west-EW", 0, "max")),
+                task=t.UTurn(target_lane_index=0),
```

Comment:
* The map should have a connection between the 2 edges so that the mission planner still can plan the route.
* If a uturn task is specified, then the route should cut early. Otherwise, there should a default uturn if the route requires it. (low priority)